contrib/lib/src/templates/mod.rs

```diff
@@ -146,6 +146,7 @@ use rocket::http::{ContentType, Status};
 const DEFAULT_TEMPLATE_DIR: &str = "templates";
 
 /// Responder that renders a dynamic template.
+/// Note that this only contains metadata, not the actual rendered template.
```

Comment:
I think this needs another line break before it, otherwise it will be merged with the line before. This can be confirmed with `cargo doc -p rocket_contrib --open`; the short description should look the same as it is now on the very bottom of the page at <https://api.rocket.rs/v0.4/rocket_contrib/templates/index.html>.

Reply:
Ah, yeah: https://github.com/SergioBenitez/Rocket/pull/1625/commits/73b95086f6bc1881b418936fdbd29174ffc7b584